python/pyspark/pandas/tests/test_expanding.py

```diff
@@ -26,37 +26,33 @@
 
 
 class ExpandingTest(PandasOnSparkTestCase, TestUtils):
-    def _test_expanding_func(self, f):
```

Comment:
what about make `f` here accept both `str` and `callable`?
Then when we need to pass some parameter into the function, it could be a `lambda`;  (but it seems that no such case right now?)
otherwise, still keep existing logic.

also cc @itholic

Reply:
> Then when we need to pass some parameter into the function, it could be a lambda; (but it seems that no such case right now?)

This refactor for https://github.com/apache/spark/commit/e22ee1c80279c5d23125c44327cd1dd58f5a592a#diff-5c57c0021cbc68b7ae93944f93d15a25dd7b7256b2e6ea4b0a141c694fea1ee7R79

> what about make f here accept both str and function?

This is ok